### DISCUSSION TITLE
_place-items-{position}_ with grids does not work in React & Gatsby

### DISCUSSION THREAD

**Discussion by Curti-s** [tailwindlabs/tailwindcss#7682] [Help] [answered]
The `place-items-{position}` does not work for React apps in tailwind CSS
Here's a link to reproduce the issue. https://codesandbox.io/s/cool-pond-qid1ds?file=/src/pages/index.js

**Comment by adamwathan** [comment, marked as answer]
Hey! This is just how CSS works, nothing Tailwind specific. When you use `place-items: start` the item is no longer stretched horizontally to fill the grid cell area, instead it’s packed to the start. Since your items have no width, the width becomes 0 and they become invisible.

If you give them a width you can see what is actually happening:

https://play.tailwindcss.com/w1JOTqXhXd

**Reply by Curti-s** [reply]
Hi @adamwathan , many thanks for pointing this out. I truly appreciate tailwind. Asante
